Rakefile

```diff
@@ -10,11 +10,12 @@ RSpec::Core::RakeTask.new(:ci) do |t|
 
   gemfile = File.basename(ENV.fetch("BUNDLE_GEMFILE", ""), ".gemfile")
 
-  if gemfile.empty? || gemfile == "Gemfile"
-    output_prefix = "default"
-  else
-    output_prefix = gemfile
-  end
+  output_prefix =
+    if gemfile.empty? || gemfile == "Gemfile"
+      "default"
+    else
+      gemfile
+    end
```

Comment:
This is _better_ but I still prefer:

```ruby
output_prefix = if gemfile.empty? || gemfile == "Gemfile"
  "default"
else
  gemfile
end
```

Reply:
@rkrage I reverted my changes and updated `rubocop_todo.yml`